src/routingComps/CreateProject.js

```diff
@@ -63,17 +60,19 @@ export default class newsfeeds extends Component {
 			description: this.refs.description.value,
 			projectPoints: this.refs.points.value,
 			dueDate: this.refs.dueDate.value,
-			isGroup: this.state.checked,
+			isGroup: isGroup,
 			groupSize:this.refs.numberOfTeam.value
```

Comment:
what is the difference between participantsNumber and grouSize?

Reply:
groupSize = max number of team members.
ParticipantsNumber = max number of students can apply.

Reply:
Can you mention that in the comments please?

Reply:
Done.